.github/workflows/upgrade_downgrade_test.yml

```diff
@@ -0,0 +1,189 @@
+name: Upgrade Downgrade Testing
```

Comment:
Should the name also include Query Serving since this test only runs the Query serving upgrade downgrade tests?

Reply:
Yes it should

Reply:
This is fixed via 903b2c3